src/api/orderApi.ts

```diff
@@ -0,0 +1,43 @@
+import RestApi from './restApi';
+
+export interface OrderApi {
+  orderByMarket: (body: any) => Promise<{[key: string]: any}>;
+  orderByPending: (body: any) => Promise<{[key: string]: any}>;
+}
+
+export class RestOrderApi extends RestApi implements OrderApi {
+  orderByMarket = (body: any) => this.fireAndForget('/Orders/market', body);
```

Comment:
i believe `post`, not `fireAndForget` 'cause we're expecting errors here

Reply:
nope, it returns just a string, so we should use .res()